src/AWS.Deploy.Recipes/CdkTemplates/BlazorWasm/AppStack.cs

```diff
@@ -48,7 +48,6 @@ private void CustomizeCDKProps(CustomizePropsEventArgs<Recipe> evnt)
             {
                 if (evnt.Props is DistributionProps props)
                 {
```

Comment:
nit: Can you leave the write line and comment the entire block like we are doing in other templates?

Reply:
Done